### PR TITLE
[WIP] Add setting to disable cohorts statsd counters (fixes #9)

### LINKED ISSUE #9: Remove absearch.cohorts.locale.* metrics
We need to stop forwarding these metrics, or send them to a dedicated Graphite host like location does.  The volume is overwhelming our metrics service.

### PR DESCRIPTION
**Pull request by leplatrem**
* [x] add setting to disable statsd
* [x] add setting `[statsd] count_cohorts_enabled ` disabled by default

### CODE REVIEW

File: absearch/server.py [hunk 1]
```diff
@@ -134,20 +137,29 @@ def handle_500_error(code):
 
 @app.route(PATH)
 def add_user_to_cohort(**kw):
-
-    with app._statsd.timer('add_user_to_cohort'):
+    if app._statsd:
+        with app._statsd.timer('add_user_to_cohort'):
+            res = app.settings.get(**kw)
```
Comment (Natim) [inline absearch/server.py]: Please use timer.start and timer.stop instead as I did here: https://github.com/mozilla-services/syncto/pull/18/files#diff-3af1fdbac63877c891e6a68862dcf289R61

File: absearch/server.py [hunk 2]
```diff
@@ -134,20 +137,29 @@ def handle_500_error(code):
...
+    else:
         res = app.settings.get(**kw)
+
+    if app._statsd and app._config['statsd']['count_cohorts_enabled']:
         cohort = res.get('cohort', 'default')
         cohort = '.'.join(['cohorts', kw['locale'],
```
Comment (Natim) [inline absearch/server.py]: Could you also rename the key from ``cohorts.locale.territory.cohort`` to to ``cohorts.locale_territory_cohort`` ?

File: absearch/server.py [hunk 3]
```diff
@@ -134,20 +137,29 @@ def handle_500_error(code):
...
+
+    return res
 
 
 @app.route('%s/<cohort>' % PATH)
 def get_cohort_settings(**kw):
-    with app._statsd.timer('get_cohort_settings'):
-        return app.settings.get(**kw)
+    if app._statsd:
+        with app._statsd.timer('get_cohort_settings'):
```
Comment (Natim) [inline absearch/server.py]: ditto.

File: config/absearch.ini [hunk 4]
```diff
@@ -65,9 +65,11 @@ num_retries = 1
 
 
 [statsd]
+enabled = 1
```
Comment (Natim) [inline config/absearch.ini]: Could you please use True instead of 1 ?

File: config/absearch.ini [hunk 5]
```diff
@@ -65,9 +65,11 @@ num_retries = 1
...
 host = localhost
 port = 8125
 prefix = absearch
+count_cohorts_enabled = 0
```
Comment (Natim) [inline config/absearch.ini]: Could you please use False instead of 0 ?

Comment (leplatrem) [inline config/absearch.ini]: yeah I know, but I left it as it was done elsewhere :/

Comment (Natim) [inline config/absearch.ini]: We might change elsewhere as well if that is what bothers you :+1: (we shall probably put them in lowercase though)

**Pull request closed without merging**